### PR TITLE
MRPHS-3404: Added dummy methods for new methods of VirtualMachine interface

### PR DESCRIPTION
**Pull request by nirav1007**
**Jira id**
[MRPHS-3404](https://apporbit.atlassian.net/browse/MRPHS-3404)

Problem
New methods have been added to VirtualMachine interface, thus dummy methods needs to be implemented in AWS/vm.go for build

Solution
Added dummy methods in vm.go

### CODE REVIEW

**Review by pankajtandon12 — CHANGES_REQUESTED** [review]
In Discussion.

**Comment by nirav1007** [comment]
Have removed declaration of AddDisk, RemoveDisk and Reset from VM Interface. But have kept added dummy methods in aws/vm.go as anyway it will have to be implemented.

**Review by avnish30jn — APPROVED** [review]
LGTM

File: virtualmachine/aws/vm.go [hunk 1]
```diff
@@ -383,10 +383,18 @@ func (vm *VM) ResetKeyPair() {
 	vm.KeyPair = ""
 }
 
-func (vm *VM) AddDisk() error {
-        return errors.New("TO-DO")
+// AddDisk adds disk to instance
```
Comment (pankajtandon12) [inline virtualmachine/aws/vm.go]: This code of dummy methods can be removed now. And added with implementation only.

**Review by pankajtandon12 — CHANGES_REQUESTED** [review]

**Review by pankajtandon12 — APPROVED** [review]
LGTM

**Pull request merged**